gamemodes/core/player/job/job_weapon-dealer-table.inc

```diff
@@ -0,0 +1,22 @@
+#include <YSI_Coding\y_hooks>
+
+hook OnMySQLConnected()
+{
+    static const query[] = "\
+    CREATE TABLE IF NOT EXISTS player_weaponInv \
```

Comment:
can we stick small letters, snake_case naming convention.

e.g

weapon_Inventory_player